docs/userguide/containers/dockerrepos.md

```diff
@@ -42,6 +42,7 @@ You can now commit and push your own images up to your repos on Docker Hub.
 > **Note:**
 > Your authentication credentials will be stored in the `~/.docker/config.json`
 > authentication file in your home directory.
+> For Windows users, this Docker file is stored in `$HOME/.docker/config.json`
```

Comment:
How about we re-phrase it as follows:

> Your authentication credentials will be stored in your home directory under `.docker/`, and can be accessed at the following location `$HOME/.docker/config.json`

Is that better?
If we go with `$HOME` then it seems to me ok as an OS-universal description, otherwise if we want to specify `%HOME%` then we should split again this note for Linux, Windows and OSx users.

What do you think?